lms/templates/user_dropdown.html

```diff
@@ -20,7 +20,7 @@
             <img class="menu-image" src="${profile_image_url}" alt="">
             ${username}
         </a>
-        <div role="group" aria-label="User menu" class="user-menu">
+        <div role="group" aria-label="User menu">
```

Comment:
I verified this locally and the drop down menu is now showing incorrectly on my devstack. I'm spinning up my sandbox now to double check. I think we need to make sure this change only fixes the RequireJS problem.